### ISSUE TITLE
[BUG] Maui Android .Net 7 Linker issue

### ISSUE THREAD

**Issue by epsmae** [mono/SkiaSharp#2327]
**Description**

Skia Sharp Nugets results in a linker errror in Maui with .Net 7 wen Startup Tracing is used for an Android Release.

**Expected Behavior**

Works in Debug and Release mode.

**Actual Behavior**

`Fatal error in IL Linker
  Unhandled exception. Mono.Linker.LinkerFatalErrorException: ILLink: error IL1011: Failed to write 'obj\Release\net7.0
  -android\android-x64\linked\SkiaSharp.Views.Android.dll`

**Basic Information**

- Version with issue:  2.88.3
- Last known good version:  2.88.1 only with .Net 6
- IDE:  Visual Studio
- Platform Target Frameworks: 
  - Android: Android 13 API level 33
  

**May be a workaround for someone**
If Aot and trimming is disabled [Disable Linking](https://stackoverflow.com/questions/74196304/net-maui-publishing-android-app-fails-due-to-issue-during-linking)

**Reproduction Link**

[Issue Sample Repo with whole error message](https://github.com/epsmae/SkiaSharp_Linker_Issue_Maui_Net_7)

**Comment by JozefGulaPayius** [comment]
Hi, I have the same issue. Inner exception from linker is:

System.ArgumentException: Member 'System.Int32 SKCanvasView_ignorePixelScaling' is declared in another module and needs to be imported

**Comment by bwinklesky** [comment]
I'm having an issue with is as well.

**Comment by RichardMarsh** [comment]
Same issue

**Comment by bcaceiro** [comment]
I'm having the same problem.

**Comment by mattleibow** [comment]
OK, based on some further testing and checking binlogs, we have confirmed it was the new linker step that was added in GA. This was disabled for the upcoming service release in https://github.com/dotnet/maui/pull/11096 as a result of https://github.com/dotnet/maui/issues/11069.

The solution for now is to use this in your csproj:

```xml
<AndroidLinkResources>false</AndroidLinkResources>
```

The next servicing release of .NET MAUI will have this disabled already.
